test/api/repositories/template-validate/006-ok/request.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Tady ten request už tu byl, ale folder se jmenoval `005-request` a já jsem to potřeboval přejmenovat na `006-ok`. Jinak se v něm vůbec nic nezměnilo.